rust/agents/relayer/src/msg/processor.rs

```diff
@@ -138,7 +138,10 @@ impl DirectionalNonceIterator {
     #[instrument]
     fn iterate(&mut self) {
         match self.direction {
-            NonceDirection::High => self.nonce = self.nonce.map(|n| n.saturating_add(1)),
+            NonceDirection::High => {
+                self.nonce = self.nonce.map(|n| n.saturating_add(1));
+                debug!(?self, "Iterating high nonce");
```

Comment:
What does it mean to iterate a high nonce

Reply:
the message processor uses a forward-backward strategy to find messages in the db (added by the indexer) and send them to the submitter. It stores a "high nonce" for the forward part, and a "low nonce" for the backward part. If the high nonce is being iterated, it means a new message was added to the db by the indexer, meaning we expect this message to be delivered soon. The nonces are initialized to the highest seen nonce (stored in the db), so if the relayer logs this shortly after startup, it means it's processing messages -> a sign of fast startup